portal/templates/select-scheme.njk

```diff
@@ -22,7 +21,7 @@
 
     {{ govukInsetText({
       classes: "govuk-!-margin-top-0",
-      text: "Further instructions have been sent to your email address."
+      text: "Further instructions have been sent to your email address if we have an account on record."
```

Comment:
We're safe to delete the `{% if passwordreset %}` and `{% if passwordupdated %}` sections of this `select-scheme.njk` template. It's never rendered with the `passwordreset` or `passwordupdated` params so they're always false